### ISSUE TITLE
Expose property doesn't work when requiring a module name

### ISSUE THREAD

**Issue by tleunen** [substack/node-browserify#850]
With the latest version (5.9.1), aliasing a module name doesn't work anymore. `browserify -r fs:browserify-fs` doesn't create the alias.
With an older version (4.2.3), an alias was created with the same command.

No issue with specifying a file instead of a module name.

**Comment by tleunen** [comment]
With the example, I still get the same error as before, with latest version. Is it something browserify won't support anymore? @substack 
```
browserify -r fs:browserify-fs
Error: Cannot find module 'browserify-fs' from ..
```

**Comment by slorber** [comment]
@tleunen I still have the problem too. The build fail only when using an alias/expose name

**Comment by code-guru** [comment]
I also have this problem both in cli and in code using `b.require('backbone', {expose: 'bb'})` . I am using Browsify `8.1.1`
``` bash
> browserify -r backbone:bb -o test.js
```

``` bash
Error: Cannot find module 'bb' from 'd:\project'
```

**Comment by badem4o** [comment]
+1

Using module name instead of module path is still broken.

**Comment by syadykin** [comment]
+1

Same here, import by name doesn't work with expose feature.

**Comment by substack** [comment]
Published @ElNouch's fix in 9.0.8.
